src/System.Net.Sockets.Legacy/System.Net.Sockets.Legacy.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
What's the distinction between the System.Net.Sockets.Legacy and System.Net.Sockets folders?